packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-await-in-loop */
```

Comment:
Maybe we can avoir disabling eslint here ?

Reply:
no, we can't avoid disabling eslint here, I need to have await in a for loop to make my http calls sequential (or i have to add an eslint disable for each ligne that have await in the for loop